contrib/epee/include/net/abstract_tcp_server2.h

```diff
@@ -36,27 +36,17 @@
 #define _ABSTRACT_TCP_SERVER2_H_ 
 
 
+#include <mutex>
 #include <string>
 #include <vector>
-#include <boost/noncopyable.hpp>
-#include <boost/shared_ptr.hpp>
 #include <atomic>
 #include <cassert>
 #include <map>
 #include <memory>
 
 #include <boost/asio.hpp>
 #include <boost/asio/ssl.hpp>
```

Comment:
We need to include `<boost/asio/steadytimer.hpp>` here for the timer change

Reply:
Fixed.